XouDevSpec/Classes/features/LoginScreen/ViewModel/LoginViewProtocol.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  LoginViewModelProtocol.swift
+//  XouDevSpec
+//
+//  Created by Xander Schoeman on 2020/04/27.
+//
+
+import Foundation
+
+public protocol LoginViewProtocol: class {
+    func retrieveLoginData(with: User)
```

Comment:
Consider thinking this as an event. This gets called by the view model when the data has been fetched. Past tense, it's already done. This method suggests doing that action of getting the data, I would suggest adding a d to retrieved to make it past tense

Reply:
noted.